Casks/s/sococo.rb

```diff
@@ -8,18 +8,23 @@
   homepage "https://app.sococo.com/a/download"
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url :homepage
+    regex(/window\._appVersion\s*=\s*["'](\d+(?:\.\d+)+)["']/i)
```

Comment:
```suggestion
    regex(/appVersion\s*=\s*["']v?(\d+(?:\.\d+)+)["']/i)
```

There may be something to be said for relaxing the variable name a little, so this would continue to match if upstream stops making `_appVersion` a global variable.